core/src/main/java/bisq/core/btc/setup/WalletsSetup.java

```diff
@@ -115,6 +111,7 @@
 public class WalletsSetup {
 
     public static final String PRE_SEGWIT_WALLET_BACKUP = "pre_segwit_haveno_BTC.wallet.backup";
+    private static final long WALLET_POLL_INTERVAL_SECONDS = 30;
```

Comment:
This is better named `DAEMON_POLL_INTERVAL_SECONDS` since it's polling the daemon peers.

Reply:
renamed